jetstream/experimenter.py

```diff
@@ -165,24 +168,55 @@ class ExperimentCollection:
     @classmethod
     def from_experimenter(cls, session: requests.Session = None) -> "ExperimentCollection":
         session = session or requests.Session()
-        legacy_experiments_json = session.get(cls.EXPERIMENTER_API_URL_V1).json()
-        legacy_experiments = []
 
-        for experiment in legacy_experiments_json:
-            if experiment["type"] != "rapid":
-                try:
-                    legacy_experiments.append(ExperimentV1.from_dict(experiment).to_experiment())
-                except Exception as e:
-                    logger.exception(str(e), exc_info=e, extra={"experiment": experiment["slug"]})
+        v1_retries = 0
+        success_v1 = False
+        v6_retries = 0
+        success_v6 = False
 
-        nimbus_experiments_json = session.get(cls.EXPERIMENTER_API_URL_V6).json()
-        nimbus_experiments = []
-
-        for experiment in nimbus_experiments_json:
+        while v1_retries < cls.MAX_RETRIES and not success_v1:
```

Comment:
This is a little annoying. Sometimes Experimenter just returns a 200 status code but an empty body resulting in an exception being thrown. Since it's a success status code, using `requests` built in retry strategy doesn't work. So I ended up implementing something myself here

Reply:
> Sometimes Experimenter just returns a 200 status code but an empty body
💀